.better-commits.json

```diff
@@ -88,16 +88,20 @@
                 "label": "app"
             },
             {
-                "value": "shared",
-                "label": "shared"
+                "value": "frontend",
+                "label": "frontend"
             },
             {
-                "value": "server",
-                "label": "server"
+                "value": "backend",
+                "label": "backend"
             },
             {
-                "value": "tools",
```

Comment:
let's keep it, better-commit itself is a tool no?

Reply:
Good point. I've added devops as a type that would include anything ci/cd, tools.. I think that makes sense, no?

Reply:
no I think tools is fine. there is a commit type above for ci and I think it's enough.